warp10/src/main/java/io/warp10/continuum/ingress/Ingress.java

```diff
@@ -742,7 +748,11 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
           }
           
           try {
-            encoder = GTSHelper.parse(lastencoder, line, extraLabels, now, maxValueSize, false);
+            if (this.acceptAttributes) {
+              encoder = GTSHelper.parse(lastencoder, line, extraLabels, now, maxValueSize, true);
```

Comment:
encoder = GTSHelper.parse(lastencoder, line, extraLabels, now, maxValueSize, this.acceptAttributes);

// with that you can remove the if/else

Reply:
good idea 👍  

fixed and squashed